omod-common/src/test/java/org/openmrs/module/webservices/rest/doc/SwaggerSpecificationCreatorTest.java

```diff
@@ -0,0 +1,120 @@
+/**
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ * http://license.openmrs.org
+ *
+ * Software distributed under the License is distributed on an "AS IS"
+ * basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+ * License for the specific language governing rights and limitations
+ * under the License.
+ *
+ * Copyright (C) OpenMRS, LLC.  All Rights Reserved.
+ */
+package org.openmrs.module.webservices.rest.doc;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.openmrs.GlobalProperty;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.webservices.docs.swagger.SwaggerSpecificationCreator;
+import org.openmrs.module.webservices.rest.web.RestConstants;
+import org.openmrs.module.webservices.rest.web.api.RestService;
+import org.openmrs.web.test.BaseModuleWebContextSensitiveTest;
+
+import java.sql.*;
+import java.util.HashMap;
+import java.util.Map;
+
+public class SwaggerSpecificationCreatorTest extends BaseModuleWebContextSensitiveTest {
+	
+	Map<String, Integer> beforeCounts;
+	
+	public Map<String, Integer> getRowCounts() throws Exception {
+		Map<String, Integer> ret = new HashMap<String, Integer>();
+		
+		Connection con = this.getConnection();
+		DatabaseMetaData metaData = con.getMetaData();
+		
+		ResultSet rs = metaData.getTables(null, "PUBLIC", "%", null);
+		while (rs.next()) {
+			String tableName = rs.getString(3);
```

Comment:
You can get table row count from the DatabaseConnection object:

`new DatabaseConnection(getConnection()).getRowCount(tableName)`